backend/tests/v5/unit/services/mailer/mailer.test.js

```diff
@@ -44,26 +44,43 @@ const testSendEmail = () => {
 		const mailerConfig = { ...config.mail };
 
 		beforeEach(() => {
-			config.mail = mailerConfig;
+			config.mail = { ...mailerConfig };
 		});
 
 		test('should fail if config.mail.sender is not set', async () => {
 			Mailer.reset();
```

Comment:
short answer is yes.

this call resets the promise. I call it before to ensure that a previous test has not already done any operation that would've initialised the mailer, and it is called after to ensure it resets the state of the test to make sure it doesn't cause another test to fail because of this test.

Initially I wanted to put it in beforeEach, but I needed a test that would pick up the init without triggering it for code coverage, hence the state of this.